intra_process_demo/README.md

```diff
@@ -75,7 +75,7 @@ ros2 run intra_process_demo camera_node
 This starts the `watermarked_node` ROS 2 node which subscribes to raw images from ROS 2 topic `/image`, overlays both **process ID number** and **message address** on top of the image visually and publishes to ROS 2 topic `/watermarked_image`.
```

Comment:
@Trushant-Adeshara-UM i think we need to change this?

```suggestion
This starts the `watermark_node` ROS 2 node which subscribes to raw images from ROS 2 topic `/image`, overlays both **process ID number** and **message address** on top of the image visually and publishes to ROS 2 topic `/watermarked_image`.
```

Reply:
I get it now, I have updated the PR.